di-demo/client/src/app/AppRoutes.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Homepage from '../components/Homepage/Homepage';
```

Comment:
Volgens mij zou je hier dus de webpack alias kunnen gebruiken, dus '@components/Homepage/Homepage'
Ik ben zelf wel voorstander van een index.ts in elke component folder zodat je '@components/Homepage' kunt gebruiken.

Reply:
lijkt me mooier, maar het werkt niet bij mij: Can't resolve '@components/Homepage' in '/usr/src/app/src/app'